src/main/webapp/app/exercises/modeling/manage/example-modeling/example-modeling-submission.component.html

```diff
@@ -39,22 +39,49 @@ <h2 jhiTranslate="artemisApp.modelingExercise.exampleSubmissionForModelingExerci
         </div>
     </div>
 
-    <div class="col-12 text-end mb-2" *ngIf="readOnly">
-        <button (click)="readAndUnderstood()" class="btn btn-success">
-            <fa-icon [icon]="'save'"></fa-icon>
-            <span jhiTranslate="artemisApp.exampleSubmission.readAndUnderstood">I have viewed and understood the example</span>
-        </button>
-    </div>
+    <div class="row">
+        <div class="col-9">
+            <span class="pe-2" style="font-weight: bolder">Legend: </span>
+            <div *ngFor="let legendElement of legend.slice(0, 3)" class="pe-5" style="display: inline-block">
+                <span>{{ legendElement.text | artemisTranslate }}</span>
+                <fa-layers>
+                    <fa-icon style="color: #e9ecef" [icon]="'circle'" size="2x"></fa-icon>
+                    <fa-icon [icon]="legendElement.icon" [styles]="{ width: '29px', color: legendElement.color, 'font-size': legendElement.size }" transform="shrink-6"></fa-icon>
+                </fa-layers>
+            </div>
 
-    <div class="col-12 text-end mb-2" *ngIf="toComplete">
-        <button
-            class="btn btn-primary px-5"
-            (click)="checkAssessment()"
-            [disabled]="!feedbacks || !assessmentsAreValid"
-            jhiTranslate="artemisApp.exampleSubmission.submitAssessment"
-        >
-            Submit Assessment
-        </button>
+            <!-- extra logic for the display warning icon -->
+            <div class="pe-5" style="display: inline-block">
+                <span>{{ legend[3].text | artemisTranslate }}</span>
+                <fa-layers>
+                    <fa-icon style="color: #e9ecef" [icon]="'circle'" size="2x"></fa-icon>
+                    <fa-icon
+                        [icon]="legend[3].icon"
+                        [styles]="{ color: '#ffc800', width: '29px', 'margin-top': '-9px', 'font-size': legend[3].size }"
```

Comment:
is it possible to extract these inline styles into some other scss file? or you can leave it as it is. I just wonder.

Reply:
I agree..., or, I am also wondering :D

Reply:
Thank you for the suggestion @imstreetspirit, I tried but I had issues when using the `[classes]` features, as documented here: https://github.com/FortAwesome/angular-fontawesome/blob/master/docs/usage/features.md

For this reason I will leave it inlined 👼